protocols/ntree/simulation.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func init() {
+	sda.RegisterNewProtocol("CoSiNtree", func(node *sda.Node) (sda.ProtocolInstance, error) { return NewProtocol(node) })
```

Comment:
Please avoid these function-registrations and use
```
sda.RegisterNewProtocol("CoSiNtree", NewProtocol(node))
```

The same goes for the other occurrences. The simpler, the better.